config/locales/mailers/en.yml

```
@@ -492,6 +492,19 @@ en:
         other: We regret to inform you that your request for %{count} new invitations cannot be fulfilled at this time.
       reason: 'Your request was:'
       subject: "[%{app_name}] Additional invitation request declined"
+    prompter_notification:
+      a_response_to_your_prompt:
+        collection:
+          html: A response to your prompt has been posted in the %{collection_link} collection at the Archive of Our Own!
+          text: A response to your prompt has been posted in the "%{collection_title}" collection (%{collection_url}) at the Archive of Our Own!
```

Perhaps it's out of scope for now, but the the preposition for referencing AO3 doesn't sound right
```suggestion
          html: A response to your prompt has been posted in the %{collection_link} collection on the Archive of Our Own!
          text: A response to your prompt has been posted in the "%{collection_title}" collection (%{collection_url}) on the Archive of Our Own!
```

---

Not sure about editing it, so I left it for now